core/templates/dev/head/domain/exploration/InteractionObjectFactory.js

```diff
@@ -48,7 +58,10 @@ oppia.factory('InteractionObjectFactory', [
       confirmed_unclassified_answers: this.confirmedUnclassifiedAnswers,
       customization_args: this.customizationArgs,
       default_outcome: this.defaultOutcome,
-      fallbacks: this.fallbacks,
+      fallbacks: this.fallbacks === undefined ? undefined :
```

Comment:
I was wondering about that too and just assumed there were cases where it's supposed to be undefined. I'll try and see what's wrong.